src/groupeddataframe/grouping.jl

```diff
@@ -906,7 +906,7 @@ function _combine_with_first!(first::Union{AbstractDataFrame,
             local newcols
             let i = i, j = j, outcols=outcols, rows=rows # Workaround for julia#15276
                 newcols = ntuple(length(outcols)) do k
-                    S = eltype(rows[k])
+                    S = eltype(rows isa AbstractDataFrame ? rows[!, k] : rows[k])
```

Comment:
```suggestion
                    S = eltype(getproperty(rows, k))
```

Reply:
`k` is not a `Symbol` so we cannot use `getproperty` here